auto/sockets

```diff
@@ -158,6 +158,61 @@ nxt_feature_test="#include <stdio.h>
                   }"
 . auto/feature
 
```

Comment:
Add empty line. Simply because there are 2 blank lines between all other checks in this file.

Reply:
done

Reply:
I was wrong, sorry. 2 empty lines required before feature test, not result check. Please revert to 1 empty line here.

Reply:
done